tox.ini

```
@@ -73,10 +73,10 @@ envlist =
 deps =
     -r test-requirements.txt
 
-    django-{1.11,2.0,2.1,2.2,3.0}: djangorestframework>=3.0.0,<4.0.0
-    py3.7-django-{1.11,2.0,2.1,2.2,3.0}: channels>2
-    py3.7-django-{1.11,2.0,2.1,2.2,3.0}: pytest-asyncio==0.10.0
-    {py2.7,py3.7}-django-{1.11,2.2,3.0}: psycopg2-binary
+    django-{1.11,2.0,2.1,2.2,3.0,dev}: djangorestframework>=3.0.0,<4.0.0
+    {py3.7,py3.8}-django-{1.11,2.0,2.1,2.2,3.0,dev}: channels>2
+    {py3.8,py3.7}-django-{1.11,2.0,2.1,2.2,3.0,dev}: pytest-asyncio==0.10.0
```

```suggestion
    {py3.7,py3.8}-django-{1.11,2.0,2.1,2.2,3.0,dev}: pytest-asyncio==0.10.0
```

Keep them sorted for our sanity?